Week-1/CattyBoard Assignment/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/font-awesome-line-awesome/css/all.min.css">
+	<link rel="stylesheet" type="text/css" href="css/stylesheet.css"/>
+</head>
+<body>
+    <div class="sidenav">
+        <span><i class="las la-bars"></i></span>
+        <span><i class="las la-home"></i></span>
+        <span><i class="las la-search"></i></span>
+        <span><i class="las la-volume-up"></i></span>
+        <span><i class="las la-user"></i></span>
+        <span><i class="las la-spotify"></i></span>
+        <span><i class="las la-cog"></i></span>
+        <span><i class="las la-soundcloud"></i></span>
+        
+      </div>
+
+
+    <div class="container">
+        <div class="flex-container" id="playlist">
+
+                <img id="mainimg" src="images/cattyboard.jpg" alt="Cattyboard 100 chart"/>
+                <div id="songname">
+                    <p id="heading">CattyBoard Top 100 Single Charts (11.06.36)</p>
+                    <p id="artist">Unknown Artist</p>
+                    <p id="description">2016.Charts.100 songs                    
+                    </p>
+                    <p id="playlist-icons">
+                        <i class="las la-play"> </i>Play all
+                        <i class="las la-plus"></i>Add to
+                        <i class="las la-ellipsis-h"></i>More
+                    </p>
+
+                </div>
+
+        </div>
+        <div id="songs">
+            <table>
+                <tr>
+                  <td>1.One Dance</td>
+                  <td>Crake feat CatKid & Cyla</td>
+                  <td>2:54</td>
+                  <td><a href="#">CATTY CLOUD SYNC</a></td>
+                </tr>
+                <tr>
+                  <td>2.Panda</td>
+                  <td>Cattee</td>
+                  <td>4:06</td>
+                  <td><a href="#">CATTY CLOUD SYNC</a></td>
+                </tr>
+                <tr>
+                  <td>3.Can't Stop the Feeling!</td>
+                  <td>Catin Cimberlake</td>
+                  <td>3:56</td>
+                  <td><a href="#">CATTY CLOUD SYNC</a></td>
+                </tr>
+                <tr>
+                  <td>4.Work From Home</td>
+                  <td>Cat Harmony feat Colla</td>
+                  <td>3:34</td>
+                  <td><a href="#">CATTY CLOUD SYNC</a></td>
+                </tr>
+              </table>
+        </div>
+        <div id="footersong">
```

Comment:
plz follow bem naming conventions for class names u can refer http://getbem.com/naming/